package/MDAnalysis/analysis/align.py

```diff
@@ -1070,7 +1069,7 @@ def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
        Biopython is now an optional dependency which this method requires.
 
     """
-    if not HAS_BIOPYTHON:
+    if biopython is None:
```

Comment:
The semantics of `if not HAS_BIOPYTHON` are clearer.

The following is extra code
```python
def has_optional_dependency(mod):
    return mod is not None
```
but may be more readable:
```python
from MDAnalysis.lib.util import optional_import, has_optional_dependency

if not has_optional_dependency(biopython):
   # do the thing when we don't have biopython
```